cloudomate/util/captchasolver.py

```diff
@@ -0,0 +1,153 @@
+import json, requests
+from time import sleep
+import os
+import base64
+
+"""
+use this already generated API key for testing: fd58e13e22604e820052b44611d61d6c
+
+Usage: 
+
+Feed Anti Captcha account API key to solver
+c_solver = captchaSolver("fd58e13e22604e820052b44611d61d6c")
+
+Find out how much money us left
+temp = c_solver.getBalance()
+
+Provide captcha image full path to solver method to get solution
+solution = c_solver.solveCaptchaTextCaseSensitive("/home/testm2/Desktop/testcscr/php/pyhton/cap2.jpg")
+print(str(solution))
+"""
+
+class captchaSolver():
+
+    __clientKey = "not set"
+         
+    def __init__(self, cKey):
```

Comment:
Please adopt a single, coherent style for variable/function naming. In Cloudomate, we're naming variables using underscores. So instead of `testVariable` we write `testVariable`. Please change this in these two classes to make it consistent with the rest of the code base.